### PR TITLE
New technomancer origin by Benblu

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request
Adds a new origin restricted to technomancers, giving some nice combat stats, and the IH survivor perk. All code made by Beblu, PR-ed it at his request.

## Why It's Good For The Game
New origin inspired by lore. Another illusion of technomancer content.

## Changelog
:cl:
add: New combat focused origin for technomancers
/:cl:

### CODE REVIEW

**Comment by Benblu** [comment]
Can confirm i wrote this, please shoot me if it's shit and not Shown.

**Comment by Sparrowbird611** [comment]
I think faction exclusive origins are retarded and from reading it I can't imagine why you would restrict this to only technos, since it seems perfectly applicable to any random sap. I think you should make it usable by everyone.

 More importantly though, it shouldn't be giving the survivor perk. If you don't want to be affected by death so badly, take nihilist.

File: code/datums/setup_option/backgrounds/origin.dm [hunk 1]
```diff
@@ -218,3 +218,22 @@
...
+	stat_modifiers = list(
+		STAT_ROB = 5,
+		STAT_TGH = 10,
+		STAT_BIO = -10,
+		STAT_MEC = -5,
+		STAT_VIG = 10,
+		STAT_COG = -10
+	)
+
+	restricted_jobs = list(/datum/job/captain, /datum/job/hop, /datum/job/chaplain, /datum/job/merchant, /datum/job/cmo, /datum/job/rd, /datum/job/ihc)
```
Comment (Firefox13) [inline code/datums/setup_option/backgrounds/origin.dm:236]: изображение

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Putting a *profession* perk into origin isn't only cringe but putting a perk into origin will wildly throw things off-balance, too. Remove it.

File: code/datums/setup_option/backgrounds/origin.dm (hunk 1 above)
Comment (TheShown911) [reply code/datums/setup_option/backgrounds/origin.dm:236]: Restricts *to* means only technos can take it.

**Comment by TheShown911** [comment]
> Putting a _profession_ perk into origin isn't only cringe but putting a perk into origin will wildly throw things off-balance, too. Remove it.

Okay dokey

File: code/datums/setup_option/backgrounds/origin.dm [hunk 2]
```diff
@@ -218,3 +218,22 @@
...
+	restricted_depts = IRONHAMMER | MEDICAL | SCIENCE | CHURCH | GUILD | CIVILIAN | SERVICE
+
+	perks = list(PERK_SURVIVOR)
```
Comment (TheShown911) [inline code/datums/setup_option/backgrounds/origin.dm:239]:
```suggestion

```

File: code/datums/setup_option/backgrounds/origin.dm (hunk 1 above)
Comment (Wouju) [reply code/datums/setup_option/backgrounds/origin.dm:236]: just to make sure, does the restricted jobs list also allow civ jobs (such as club) and vags to take the origin or only technos?

Comment (TheShown911) [reply code/datums/setup_option/backgrounds/origin.dm:236]: looks_good

**Review by Firefox13 — APPROVED** [review]

File: code/datums/setup_option/backgrounds/origin.dm (hunk 1 above)
Comment (Wouju) [reply code/datums/setup_option/backgrounds/origin.dm:236]: yooo AI with the techno origin??? 😳

**Pull request merged**